### ISSUE TITLE
Inconsistent highlighting for C# sub-classes or multiple classes in one file

### ISSUE THREAD

**Issue by geh24** [FortuneN/FineCodeCoverage#430]
### Installed product versions
- Visual Studio: 17.10.3 Community
- Fine Code Coverage 1.1.125

### Description
1) The highlighting is inconsistent if a class is a C# sub-class
2) The hightlighting is missing if the order of classe within a file ar changed

### Steps to recreate
1. Please see my repo https://github.com/geh24/FCC
It is a really small test-project and just run the Unit-Test
and in MulHelper2.multiply all lines are highlighted correctly
but in MulHelper1.multiply only the first line is highlighted.

I except all three lines 24-26 to be highlighted.
![FccMath01](https://github.com/FortuneN/FineCodeCoverage/assets/104834939/46330728-7ced-482c-b174-78683274e820)
I already checked the Cobertura.xml file and in that file all lines 24-26 are marked as hits.

2. **If the order of the classes FCCMath and MulHelper2 are changed**,
please see the branch ClassExchange (just run the UnitTest)
and then no line of MulHelper2 is highlighted even it is run (a breakpoint will be hit)
![FccMath02](https://github.com/FortuneN/FineCodeCoverage/assets/104834939/cb83e599-b2de-4f8e-a1e4-e6bda5dcdea1)

### Expected behavior
All lines of code that are run during a unit-test should be highlighted,
independently whether it is a sub-class or the order of classes within a file.

I will attach screenshots and Cobertura.xml files for both cases.

### Side Notes
- Please see my github-repo https://github.com/geh24/FCC
- See the attached screenshots
- It helps to attach the log (send it to my mailbox if you need todo so privately : fortunengwenya@gmail.com)
I do not know what logfiles i can send you, i tried to send you the Cobertura.xml files, but it seems to me
they are blocked by github, but you can easily reproduce them by running the Unit-Test.
- Please consider giving this project a star - already done
- Please consider buying me a coffee
I have no idea how much is a coffee in Johannesburg,  i just send you 20€ via PayPal, in Munich you would
get 3-5 Coffees for that money, enjoy them

I hope to hear from you
best regards
Gerhard

**Comment by tonyhallett** [comment]
I should be able to look at this tomorrow

**Comment by tonyhallett** [comment]
fixed.  Thanks for reporting the issue.  Will be available on the marketplace shortly.

**Comment by geh24** [comment]
Hi Tony,
i just ried it with a really complicated EF-Core prject and the fix works perfect.

thank you
Gerhard
